Resources/views/Form/Type/sonata_type_model_autocomplete.html.twig

```diff
@@ -34,7 +35,8 @@ file that was distributed with this source code.
                 placeholder: '{{ placeholder ?: allowClearPlaceholder }}', // allowClear needs placeholder to work properly
                 allowClear: {{ required ? 'false' : 'true' }},
                 enable: {{ disabled ? 'false' : 'true' }},
-                readonly: {{ read_only is defined and read_only or attr.readonly is defined and attr.readonly ? 'true' : 'false' }},
+                readonly: {{ read_only is defined and read_only or attr.readonly is defined and attr.readonly ? 'true' : 'false' }},  {# NEXT_MAJOR: remove #}
```

Comment:
Conditional precedence works as expected here?

It should work like this: (read_only is defined and read_only) or (attr.readonly is defined and attr.readonly)